src/main/java/duke/command/ToDoCommand.java

```diff
@@ -0,0 +1,59 @@
+package duke.command;
+
+import duke.Ui;
+import duke.io.Storage;
+import duke.io.TaskList;
+import duke.task.Todo;
+
+/**
+ * ToDo command type.
+ * Create todo task and add into task arraylist.
+ *
+ * @author Galvin Leow Wen Yuan
+ * @author A0200204J
+ * @version v1.0
+ */
+public class ToDoCommand extends Command {
```

Comment:
Perhaps the class name could be TodoCommand with d in small caps since your todo class has d in small caps. This ill make your code more standardised.